packages/core/src/browser/quick-view-service.ts

```diff
@@ -28,11 +28,9 @@ export interface QuickViewItem {
 
 @injectable()
 export class QuickViewService implements QuickOpenModel, QuickOpenHandler, QuickOpenContribution {
-
+    private hiddenItems: (QuickOpenItem & { when?: string })[] = [];
```

Comment:
Can you explain what the `hiddenItems` are used for?

Reply:
As per @RomanNikitenko code review, I added functionality for hiding/showing items from Quick Open view. Currently if you remove items from quick open list you cannot return them and you do not have original list saved - you need to recreate entire list which I didn't like as a solution - I thought this solution is least intrusive. 

> **_6. VS Code allows to display Refactor Preview panel if refactoring is in progress, I'm not able to display the panel after canceling refactoring for example. Theia displays empty panel with the corresponding message:_**
> 
> > a) Current system functionality (as far as I can tell) is disposing of widget on closing of tab (on X button) - so I cannot display workspaceEdits after it.
> > b) I fixed so that "Refactor Preview" in displayed in quick open view only when "Refactor Preview" bottom pane is displayed. otherwise it is hidden from quick open view.
> > When Refactor Preview pane is displayed:
> > ![image](https://user-images.githubusercontent.com/71069170/97596877-decff600-1a0d-11eb-86bf-84c7ad69b1d3.png)
> 
> When Refactor Preview pane is not displayed:
> ![image](https://user-images.githubusercontent.com/71069170/97596841-d37cca80-1a0d-11eb-913b-a494c7bc8fca.png)